hw3/03-react/game-of-thrones/src/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Route, Routes } from "react-router-dom";
+import "./App.css";
+import Home from "./views/Home";
+import Search from "./views/Search";
+import Houses from "./views/Houses";
+
+function Navbar() {
+  return (
+    <div>
+      <nav>
+        <ul className="Navbar">
+          <li>
```

Comment:
You can consider using react-bootstrap that can reduce working on css file a lot and also easy to use.

Reply:
Thanks for the suggestion. I replaced my css styles with bootstrap. It was easy to use as you mentioned.